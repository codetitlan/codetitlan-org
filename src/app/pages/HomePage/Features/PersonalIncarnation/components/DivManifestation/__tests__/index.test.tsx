import * as React from 'react';
import { render } from '@testing-library/react';

import { DivManifestation } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<DivManifestation  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<DivManifestation />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
