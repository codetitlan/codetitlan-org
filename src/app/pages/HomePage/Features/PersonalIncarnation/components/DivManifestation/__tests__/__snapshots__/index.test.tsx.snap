// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DivManifestation  /> should match snapshot 1`] = `
<div
  class="DivManifestation__Div-sc-1oiyuw8-0"
>
  DivManifestation
</div>
`;
